app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsPresenter.java

```diff
@@ -36,11 +36,6 @@ public void attach() {
     }
 
     @Override
-    public void detach() {
```

Comment:
Same here. If detach is removed, it opens new windows to numerous memory leaks. Please don't remove detach